core/src/main/scala/chisel3/choice/package.scala

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.experimental.{BaseModule, SourceInfo}
+import chisel3.util.simpleClassName
+
+/** This package contains Chisel language definitions for describing configuration options and their accepted values.
+  */
+package object choice {
+
+  /** An option group declaration. Specifies a container grouping values for some design configuration parameter.
+    *
+    * Example:
+    * ```
+    * import chisel3.option.{Group, Case}
+    * object Platform extends Group {
+    *   object FPGA extends Case
+    *   object ASIC extends Case
+    * }
+    * ```
```

Comment:
```suggestion
    * @example
    * {{{
    * import chisel3.option.{Group, Case}
    * object Platform extends Group {
    *   object FPGA extends Case
    *   object ASIC extends Case
    * }
    * }}}
```
This is the way examples are done in Scala 2 ScalaDoc (I think Scala 3 might use markdown, alas).